mevolib/fetch/BioSeqs.py

```diff
@@ -273,84 +217,77 @@ def from_seqfile (cls, seqfile, fileformat):
             # created that will raise an error when writing it in a GENBANK
             # file. Thus, we change that alphabet to a more specific one,
             # checking if it is a DNA or a protein sequence
-            if (isinstance(record.seq.alphabet,
-                            Alphabet.SingleLetterAlphabet)):
+            if isinstance(record.seq.alphabet,
+                            Alphabet.SingleLetterAlphabet):
                 record.seq.alphabet = Alphabet.IUPAC.ExtendedIUPACDNA()
-                if (not Alphabet._verify_alphabet(record.seq)):
+                if not Alphabet._verify_alphabet(record.seq):
                     record.seq.alphabet = Alphabet.IUPAC.ExtendedIUPACProtein()
             seq_dict[record.id] = record
         date_time = datetime.now().strftime('%Y/%m/%d %H:%M:%S')
         report = [(date_time, 'local', filepath, fileformat)]
-        return (cls(seq_dict, report))
+        return cls(seq_dict, report)
+
 
     @classmethod
-    def from_entrez (cls, email, entrez_db, query, max_fetch = sys.maxsize):
-        """
-        Create a BioSeqs object fetching the sequences that matches the query
-        at the provided database from NCBI's Entrez.
-
-        Arguments:
-            email  (string)
-                E-mail required by Bio.Entrez.
-            entrez_db  (string)
-                NCBI's Entrez database.
-            query  (string)
-                Query to fetch the desired information.
-            max_fetch  (Optional[int])
-                Maximum number of sequences to fetch.
-
-        * The e-mail information is considered sensible information and it won't
-        be saved in any public or private variable of the object.
+    def from_entrez (cls, email: str, entrez_db: str, query: str, max_fetch: Optional[int] = sys.maxsize):
```

Comment:
```suggestion
    def from_entrez (cls, email: str, entrez_db: str, query: str, max_fetch: int = sys.maxsize) -> BioSeqs:
```
`max_fetch` cannot be `None`, so `Optional` is not correct here.